present/slideshow.py

```diff
@@ -23,11 +28,33 @@
 )
 
 
+def render_code_block(screen, block, row):
+    # Divide the width by 3
+    column = int(screen.dimensions[1] / 3)
+    lexer = get_lexer_by_name(block.lang())
+
+    for cur_row, line in enumerate(block.padded_lines(), start=row):
+        coded_text = highlight(line, lexer, Terminal256Formatter())
+        text = ColouredText(
+            coded_text,
+            AnsiTerminalParser(),
+        )
+        screen.paint(
+            text,
+            column,
+            cur_row,
+            colour_map=text.colour_map,
+        )
```

Comment:
This is the main part of this PR. The other changes are about supporting this one and making the call in the right spot in the animation loop.